packages/wallet/frontend/src/pages/transfer/send.tsx

```diff
@@ -26,16 +26,48 @@ import {
 } from '@/utils/constants'
 import { useOnboardingContext } from '@/lib/context/onboarding'
 import { QuoteDialog } from '@/components/dialogs/QuoteDialog'
+import { assetService, Rates } from '@/lib/api/asset'
 
 type SendProps = InferGetServerSidePropsType<typeof getServerSideProps>
 
+type Asset = { assetCode: string; assetScale: number }
 const SendPage: NextPageWithLayout<SendProps> = ({ accounts }) => {
   const [openDialog, closeDialog] = useDialog()
   const [paymentPointers, setPaymentPointers] = useState<SelectOption[]>([])
   const [balance, setBalance] = useState('')
+  const [currentExchangeRates, setCurrentExchangeRates] = useState<Rates>()
+  const [selectedAsset, setSelectedAsset] = useState<Asset | null>(null)
+  const [receiverAssetCode, setReceiverAssetCode] = useState<string | null>(
+    null
+  )
+  const [convertAmount, setConvertAmount] = useState(0)
   const { isUserFirstTime, setRunOnboarding, stepIndex, setStepIndex } =
     useOnboardingContext()
   const [isToggleDisabled, setIsToggleDisabled] = useState(false)
+
+  const ExchangeRate = memo(() => {
+    if (
+      convertAmount &&
+      convertAmount !== 0 &&
+      currentExchangeRates &&
+      !('success' in currentExchangeRates) &&
+      receiverAssetCode &&
+      selectedAsset &&
+      receiverAssetCode !== selectedAsset.assetCode
+    ) {
+      return (
+        <p className="ml-2 text-sm text-green">{`Receiver Payment Pointer asset is in ${getCurrencySymbol(
+          receiverAssetCode
+        )}. Exchange Rate: ${getCurrencySymbol(
+          selectedAsset.assetCode
+        )}${convertAmount} = ${getCurrencySymbol(receiverAssetCode)}${(
+          convertAmount * currentExchangeRates.rates[receiverAssetCode]
+        ).toFixed(selectedAsset.assetScale)}`}</p>
+      )
+    } else return null
+  })
+  ExchangeRate.displayName = 'ExchangeRate'
```

Comment:
You can move this outside the `SendPage` component and pass in the props that it needs for displaying this - `receiverAssetCode`, `convertAmount`, etc...